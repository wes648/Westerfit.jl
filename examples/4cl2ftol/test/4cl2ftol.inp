4cl2ftol   @   2024-03-07T16:27:23.596
%CNTRLS
NFOLD = 3
S = 1.5
TK = 25.0
RUNmode = F
mcalc = 5
vtmax = 0
assign = RAM36
turducken = 1
Jmax = 16.5
νmin = 4.0
νmax = 23.0

%2NDORDER
A;             3028.963881355823;   1.0
B;             866.5049228604415;   1.0
C;             672.4557318078677;   1.0
Dab;         -126.68257903237395;   1.0
ϵzz;                         0.0;   0.0
ϵxx;                         0.0;   0.0
ϵyy;                        -0.0;   0.0
ϵxz;                        -0.0;   0.0
χzz;          -69.21833810747393;   1.0
χxz;                         0.0;   0.0
χxmy;         2.4180565623255466;   1.0
F;            5.3925209999999995;   0.0
ρ;          0.018425246526301408;   1.0
Vn;           224.17612807809692;   1.0
η;                           0.0;   0.0

%PARAMS N^a Nz^b (N₊^c + N₋^c) (NS)^d Sz^e Pₐ^f cos(g*α) sin(h*α) Ny^(1-δ(0,h))
%Op;                         Val;   Scl;   a;   b;   c;   d;   e;   f;   g;   h;  stg
μa;                          1.0;   0.0;   0;   1;   0;   0;   0;   0;   0;   0;   0
μa(3);                     0.008;   0.0;   0;   1;   0;   0;   0;   0;   3;   0;   0
μb;                          1.0;   0.0;   0;   0;   1;   0;   0;   0;   0;   0;   0
μb(3);                     0.008;   0.0;   0;   0;   1;   0;   0;   0;   3;   0;   0
μc(3);                    -0.034;   0.0;   0;   0;   0;   0;   0;   0;   0;   3;   0
DN;        -8.097181188832632e-6;   1.0;   4;   0;   0;   0;   0;   0;   0;   0;   1
V3N;          -0.379869340581063;   1.0;   2;   0;   0;   0;   0;   0;   0;   0;   1
V3N;                        -1.0;   1.0;   2;   0;   0;   0;   0;   0;   3;   0;  -1
