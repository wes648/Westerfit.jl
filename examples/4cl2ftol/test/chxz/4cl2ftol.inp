4cl2ftol   @   2024-03-10T15:05:34.067
%CNTRLS
NFOLD = 3
S = 1.5
TK = 25.0
RUNmode = F
mcalc = 5
vtmax = 0
assign = RAM36
turducken = 3
Jmax = 16.5
νmin = 4.0
νmax = 23.0

%2NDORDER
A;            3029.0060999036377;   1.0
B;              866.463988095683;   1.0
C;             672.4545087087876;   1.0
Dab;         -126.34201019065637;   1.0
ϵzz;                         0.0;   0.0
ϵxx;                         0.0;   0.0
ϵyy;                        -0.0;   0.0
ϵxz;                        -0.0;   0.0
χzz;          -66.30933494394648;   1.0
χxz;          23.455831878369114;   1.0
χxmy;        -0.3119130913194018;   1.0
F;            5.3925209999999995;   0.0
ρ;          0.018400080908646834;   1.0
Vn;           224.04676885617286;   1.0
η;                           0.0;   0.0

%PARAMS N^a Nz^b (N₊^c + N₋^c) (NS)^d Sz^e Pₐ^f cos(g*α) sin(h*α) Ny^(1-δ(0,h))
%Op;                         Val;   Scl;   a;   b;   c;   d;   e;   f;   g;   h;  stg
μa;                          1.0;   0.0;   0;   1;   0;   0;   0;   0;   0;   0;   0
μa(3);                     0.008;   0.0;   0;   1;   0;   0;   0;   0;   3;   0;   0
μb;                          1.0;   0.0;   0;   0;   1;   0;   0;   0;   0;   0;   0
μb(3);                     0.008;   0.0;   0;   0;   1;   0;   0;   0;   3;   0;   0
μc(3);                    -0.034;   0.0;   0;   0;   0;   0;   0;   0;   0;   3;   0
DN;        -8.193091501750014e-6;   1.0;   4;   0;   0;   0;   0;   0;   0;   0;   1
V3N;        -0.37449727891713464;   1.0;   2;   0;   0;   0;   0;   0;   0;   0;   1
V3N;                        -1.0;   1.0;   2;   0;   0;   0;   0;   0;   3;   0;  -1
