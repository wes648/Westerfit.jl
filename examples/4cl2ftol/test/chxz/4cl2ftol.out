4cl2ftol   @   2024-03-10T15:02:33.381

Control Parameters
mcalc = 5
νmax = 23.0
maxiter = 60
overwrite = true
Irrep = Ir
λlm0 = 0.0001
apology = true
Jmax = 16.5
S = 1.5
assign = RAM36
turducken = 3
REJECT = 10.0
TK = 25.0
RUNmode =  F
vtmax = 0
NFOLD = 3
νmin = 4.0
INTthres = 1.0e-5

Initial Parameters
 BK;           2314.367131921589;   1.0
 BN;           769.4802427245115;   1.0
 B⨦;           48.51225550901847;   1.0
 Dab;        -126.68113669184979;   1.0
 T⁰₀(ϵ);                    -0.0;   0.0
 T²₀(ϵ);                     0.0;   0.0
 T²₁(ϵ);                     0.0;   0.0
 T²₂(ϵ);                     0.0;   0.0
 T²₀(χ);      -69.21833704449307;   1.0
 T²₁(χ);     -10.614455552060438;   1.0
 T²₂(χ);      0.9871639213642895;   1.0
 F;            161663.7125406618;   0.0
 -2ρF;        -5957.396099849091;   1.0
 V3/2;       3.360317290238337e6;   1.0
 η;                          0.0;   0.0

 DN;        -8.097206047010131e-6;   1.0
 V3N;        -0.37986897738390585;   1.0
 V3N;                        -1.0;   1.0

Number of lines = 312

Initial RMS = 0.021820915762145798 MHz
Initial λ = 2.1820677691158377e-9

-------------------------------------

;            2314.280279880916;    -0.0869
 BN;            769.4592482783335;    -0.0210
 B⨦;            48.50237000252103;    -0.0099
 Dab;         -126.34200845491382;     0.3391
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);       -65.85309931663028;     3.3652
 T²₁(χ);      -22.289473309330553;   -11.6750
 T²₂(χ);      -0.3136584266476883;    -1.3008
 F;             161663.7125406618;      Fixed
 -2ρF;         -5949.255278154538;     8.1408
 V3/2;        3.358377830878095e6; -1939.4594
 η;                           0.0;      Fixed
 DN;        -8.193284689597111e-6;    -0.0000
 V3N;        -0.37449508754295047;     0.0054
 V3N;                        -1.0;      Fixed

Max Change; T²₂(χ), 131.7737%

-------------------------------------

After   2 Iterations:

RMS = 0.0042 MHz, log₁₀(λ) = -2.3793

                        Parameter     Change
 BK;            2314.280078477369;    -0.0002
 BN;            769.4592493070744;     0.0000
 B⨦;           48.502369655034194;    -0.0000
 Dab;         -126.34194959178814;     0.0001
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);       -66.34158362818063;    -0.4885
 T²₁(χ);       -18.87691857950182;     3.4126
 T²₂(χ);     -0.11109978574009405;     0.2026
 F;             161663.7125406618;      Fixed
 -2ρF;         -5949.237093372408;     0.0182
 V3/2;        3.358351678148198e6;   -26.1527
 η;                           0.0;      Fixed
 DN;          -8.1970516072421e-6;    -0.0000
 V3N;          -0.374500987814579;    -0.0000
 V3N;                        -1.0;      Fixed

Max Change; T²₂(χ), 64.5794%

-------------------------------------

After   3 Iterations:

RMS = 0.0042 MHz, log₁₀(λ) = -2.6931

                        Parameter     Change
 BK;           2314.2800830122196;     0.0000
 BN;            769.4592485982095;    -0.0000
 B⨦;            48.50236962053116;    -0.0000
 Dab;         -126.34201392640855;    -0.0001
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);       -66.28562373116677;     0.0560
 T²₁(χ);      -19.327235347191838;    -0.4503
 T²₂(χ);     -0.13767602290130987;    -0.0266
 F;             161663.7125406618;      Fixed
 -2ρF;         -5949.245830778245;    -0.0087
 V3/2;       3.3583804624922546e6;    28.7843
 η;                           0.0;      Fixed
 DN;        -8.192173814001648e-6;     0.0000
 V3N;         -0.3745004104530399;     0.0000
 V3N;                        -1.0;      Fixed

Max Change; T²₂(χ), 23.9211%

-------------------------------------

After   4 Iterations:

RMS = 0.0042 MHz, log₁₀(λ) = -2.3644

                        Parameter     Change
 BK;            2314.280105933934;     0.0000
 BN;             769.459248676772;     0.0000
 B⨦;            48.50236967629704;     0.0000
 Dab;         -126.34200211865921;     0.0000
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);       -66.30968537516588;    -0.0241
 T²₁(χ);      -19.147055175213637;     0.1802
 T²₂(χ);     -0.12706592048004847;     0.0106
 F;             161663.7125406618;      Fixed
 -2ρF;         -5949.245329048256;     0.0005
 V3/2;        3.358374652502866e6;    -5.8100
 η;                           0.0;      Fixed
 DN;        -8.193250550816594e-6;    -0.0000
 V3N;         -0.3744997710297158;     0.0000
 V3N;                        -1.0;      Fixed

Max Change; T²₂(χ), 7.7066%

-------------------------------------

After   5 Iterations:

RMS = 0.0042 MHz, log₁₀(λ) = -2.6375

                        Parameter     Change
 BK;           2314.2801404532115;     0.0000
 BN;            769.4592485477158;    -0.0000
 B⨦;              48.502369742057;     0.0000
 Dab;         -126.34200728760295;    -0.0000
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);       -66.30764368800658;     0.0020
 T²₁(χ);      -19.163986057940683;    -0.0169
 T²₂(χ);     -0.12806626164169166;    -0.0010
 F;             161663.7125406618;      Fixed
 -2ρF;         -5949.247615687287;    -0.0023
 V3/2;        3.358376282216626e6;     1.6297
 η;                           0.0;      Fixed
 DN;        -8.193034494255054e-6;     0.0000
 V3N;        -0.37449877910839546;     0.0000
 V3N;                        -1.0;      Fixed

Max Change; T²₂(χ), 0.7873%

-------------------------------------

After   6 Iterations:

RMS = 0.0042 MHz, log₁₀(λ) = -2.9105

                        Parameter     Change
 BK;            2314.280199363941;     0.0001
 BN;            769.4592484022353;    -0.0000
 B⨦;            48.50236984672388;     0.0000
 Dab;         -126.34201019065637;    -0.0000
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);       -66.30933494394648;    -0.0017
 T²₁(χ);      -19.151606531503944;     0.0124
 T²₂(χ);      -0.1273379863044446;     0.0007
 F;             161663.7125406618;      Fixed
 -2ρF;         -5949.250781480802;    -0.0032
 V3/2;       3.3583765771174957e6;     0.2949
 η;                           0.0;      Fixed
 DN;        -8.193091501750014e-6;    -0.0000
 V3N;        -0.37449727891713464;     0.0000
 V3N;                        -1.0;      Fixed

Max Change; T²₂(χ), 0.5687%

-------------------------------------

 It would appear step size has converged.

After   6 iterations:

Final RMS = 0.0042 MHz
Final log₁₀(λ) = -2.9105


                         Parameter                   Uncertainty
 A;            3029.0060999036377;           0.14319198812754097
 B;              866.463988095683;           0.06484680784018265
 C;             672.4545087087876;           0.06484680784018265
 Dab;         -126.34201019065637;            0.7690141195380256
 ϵzz;                         0.0;                           0.0
 ϵxx;                         0.0;                           0.0
 ϵxz;                        -0.0;                           0.0
 ϵyy;                        -0.0;                           0.0
 χzz;          -66.30933494394648;           0.20012424453274788
 χxz;          23.455831878369114;            1.5613162963352059
 χxx-χyy;     -0.3119130913194018;           0.18369601097946897
 F;            5.3925209999999995;                           0.0
 ρ;          0.018400080908646834;          1.770787038041513e-5
 V3;           224.04676885617286;           0.06690655435962425
 η;                           0.0;                           0.0
 DN;        -8.193091501750014e-6;         1.2343597318721647e-7
 V3N;        -0.37449727891713464;          0.010603164262322976
 V3N;                        -1.0;                           0.0


Journal Formatted Values
 A;                 3.02901(14)e3
 B;                 866.464(65)e0
 C;                 672.455(65)e0
 Dab;               -126.34(77)e0
 ϵzz;               0.0e0 (Fixed)
 ϵxx;               0.0e0 (Fixed)
 ϵxz;              -0.0e0 (Fixed)
 ϵyy;              -0.0e0 (Fixed)
 χzz;                 -66.31(2)e0
 χxz;                  23.5(16)e0
 χxx-χyy;           -310.0(18)e-3
 F;             5.39252e0 (Fixed)
 ρ;                   18.4(18)e-3
 V3;                224.047(67)e0
 η;                 0.0e0 (Fixed)
 DN;                 -8.19(12)e-6
 V3N;               -374.0(11)e-3
 V3N;               -1.e0 (Fixed)


 Again, sorry about the name...


